### ISSUE TITLE
SCSS: map-get() with interpolation throws error

### ISSUE THREAD

**Issue by laurencedorman** [tonyganch/gonzales-pe#93]
Hi, I'm using [csscomb](https://github.com/csscomb/csscomb.js) which depends on this project, and there appears to be an edge case which is causing problems in my code:

``` scss
height: calc(100% - #{map-get($nav-height, desktop)}); // this line throws an error
```

Sass obliges me to interpolate a variable within a calc(), and csscomb is fine with this; the problem only presents itself when I use a sass-map.

**Comment by tonyganch** [comment]
Should be fixed in `dev` with #115, will publish a release in a few hours.

**Comment by tonyganch** [comment]
Please check in `@3.2.3`.
